### ISSUE TITLE
some urls are checked twice

### ISSUE THREAD

**Issue by oupala** [linkchecker/linkchecker#72]
I don't know why but some urls are checked twice. And, no, the urls are not twice in my document, it is the same url, written only once in my document.

See this linkchecker logs:

```
URL 	`http://www.example.org/international/2012/08/27/slug'
Name	`title of the article'
Parent URL	http://blog.example.org/2012/09/04/another-slug/, line 110, col 137 (HTML) (CSS)
Real URL	http://www.example.org/international/2012/08/27/slug
Size	478B
Check time	0.955 seconds
Result	Error: 404 Not Found

URL 	`http://www.example.org/international/2012/08/27/slug'
Name	`title of the article'
Parent URL	http://blog.example.org/2012/09/04/another-slug/, line 110, col 137 (HTML) (CSS)
Real URL	http://www.example.org/international/2012/08/27/slug
Size	478B
Check time	1.080 seconds
Result	Error: 404 Not Found
```

As you can see, both checks use the same url, same line, same column.

**Comment by thezoggy** [comment]
From the output it looks like it was just following a redirect. If you pass `-r0` or set `recursionlevel=0` in your userconfig do you still see it?

**Comment by cjmayo** [comment]
A lack of information to reproduce here (not even a version number).  `-Dall` might reveal what is leading to each check.

**Comment by oupala** [comment]
I was using debian stretch, so linkchecker version was 9.3 (package version 9.3-4).

**Comment by oupala** [comment]
It seems that the bug is fixed with linkchecker 9.4.0.

Thanks for the fix.
